ui/core/components/icon_picker.tsx

```diff
@@ -80,7 +81,12 @@ export class IconPicker<ModObject, ValueType> extends Input<ModObject, ValueType
 		this.improvedAnchor2 = ia2.value!;
 		this.counterElem = ce.value!;
 
-		this.config.actionId.fillAndSet(this.rootAnchor, true, true);
+		const fillActionid = () => this.config.actionId(this.modObject)?.fillAndSet(this.rootAnchor, true, true);
+		this.config.changedEvent(this.modObject).on(() => {
+			console.log('good')
+			fillActionid()
+		})
+		fillActionid()
```

Comment:
We change this to a function so that we can key off of the `changedEvent` to update Action IDs each time